### ISSUE TITLE
Cannot set properties of undefined (setting '$')

### ISSUE THREAD

**Issue by huzaifaarain** [yajra/laravel-datatables-vite#1]
Hi @yajra , Hope you are doing well.

I recently tried Laravel 9 along with yajra/laravel-datatables:^v9.0.0. As of this version you have not talked about laravel-datatables-vite like you did in version 10, but i wanted to give it a try. Datatable is working fine as expected but I am seeing an error in the console.

![image](https://user-images.githubusercontent.com/8613679/200628158-26cd1726-49ce-46f9-9185-c415c0005c5c.png)

```js
// jquery.dataTables.mjs:59:7
this.$ = function ( sSelector, oOpts )
{
    return this.api(true).$( sSelector, oOpts );
};
```

After some R&D, if I am removing the following lines from the laravel-datatables-vite index.js file, the datatable is still working as expected but the error is then gone. Do you have any idea about that? Is it expected behavior?

```js
// Line 12 to 14
DataTable(window, window.$);
Buttons(window, window.$);
Select(window, window.$);
```

Regards

**Comment by yajra** [comment]
Nvm, I tried to do a new project from the quick starter I did and I think something needs to be updated. Will check and update the guide when I found the issue. Thanks for reporting!

**Comment by yajra** [comment]
I released v0.4.0 to patch the installation of dependencies. However, the bug seems to be in the datatables bs5 library itself per https://datatables.net/forums/discussion/comment/212403/#Comment_212403. 

The version I used when it works in the quick starter is ^1.12

```js
"datatables.net": "^1.12.1",
"datatables.net-bs5": "^1.12.1",
```

Installing now results in 1.13.1 

```js
        "node_modules/datatables.net": {
            "version": "1.13.1",
            "resolved": "https://registry.npmjs.org/datatables.net/-/datatables.net-1.13.1.tgz",
            "integrity": "sha512-cX5dDHsbVdLLYKsWOSE0MvuGUcV88zU5dZ/taK2puJV6F9Fw0CFsP3+U/kr+qpDSFOBLWISRyM4Q9wWWovPTNg==",
            "dev": true,
            "dependencies": {
                "jquery": ">=1.7"
            }
        },
        "node_modules/datatables.net-bs5": {
            "version": "1.13.1",
            "resolved": "https://registry.npmjs.org/datatables.net-bs5/-/datatables.net-bs5-1.13.1.tgz",
            "integrity": "sha512-kNmvUc23FKmK+QNpPPlNYYy+L/OQmiheRT7kTsp3cxHbtXsV7d6u/5TMxT70HH2VvlUivrgRcEtye4Mp+8mbKw==",
            "dev": true,
            "dependencies": {
                "datatables.net": ">=1.12.1",
                "jquery": ">=1.7"
            }
        },
```

Works fine with these versions:

```json
        "datatables.net": "1.12.1",
        "datatables.net-bs5": "1.12.1",
        "datatables.net-buttons": "2.2.3",
        "datatables.net-buttons-bs5": "2.2.3",
        "datatables.net-dt": "1.12.1",
        "datatables.net-select": "1.4.0",
        "datatables.net-select-bs5": "1.4.0",
```

**Comment by huzaifaarain** [comment]
Perfect, thank you so much for your time @yajra , Have a good day ahead.

**Comment by yajra** [comment]
Fixed via #2 c/o @AllanJard
